javaparser-core/src/main/java/com/github/javaparser/javadoc/description/JavadocInlineTag.java

```diff
@@ -87,7 +87,7 @@ public JavadocInlineTag(Type type, String content) {
 
     @Override
     public String toText() {
-        throw new UnsupportedOperationException();
+        return "@" + type.keyword + this.content;
```

Comment:
I think we should save the name of the tag and use it instead of keyword. Consider what happens if I invoke `fromText("{@foo something}")`

```
public static JavadocDescriptionElement fromText(String text) {
        if (!text.startsWith("{@")) {
            throw new IllegalArgumentException(String.format("Expected to start with '{@'. Text '%s'", text));
        }
        if (!text.endsWith("}")) {
            throw new IllegalArgumentException(String.format("Expected to end with '}'. Text '%s'", text));
        }
        text = text.substring(2, text.length() - 1);
        String tagName = nextWord(text);
        Type type = Type.fromName(tagName);
        String content = text.substring(tagName.length());
        return new JavadocInlineTag(type, content);
    }
```

The tagName will be `foo` and the type will be `UNKNOWN`. We should save the tagName and store it to use it in toText, otherwise:

`JavadocInlineTag.fromText("{@foo something}").toText()` would produce `{@unknown something}` instead of `{@foo something}`.

It would be very nice if you could add a test for this case.